fiftyone/server/lightning.py

```
@@ -345,8 +345,13 @@ async def _do_distinct_query(
     query: DistinctQuery,
 ):
     match = None
+    matcher = lambda v: False
```

Ruff (0.8.2)

Do not assign a `lambda` expression, use a `def`

Rewrite `matcher` as a `def`

(E731)